Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/LoadTestDataDocumentBean.java

```diff
@@ -68,8 +68,8 @@
 public class LoadTestDataDocumentBean {
     private static final Logger LOG = LoggerFactory.getLogger(LoadTestDataDocumentBean.class);
     private static final String DOCUMENT = "Document Info";
-    private static final String EVENT_CODE = "Eventcode";
-    private static final String GROWL_MESSAGE = "msgForGrowl";
+    private static final String EVENT_CODE = " an Eventcode";
+    private static final String GROWL_MESSAGE = "a msgForGrowl";
```

Comment:
Please elaborate on this.

Reply:
I do not need to update the Growl_Message because it is a identifier that there is a message but event_code is part of the message. Previously it had been a/an.